docs/src/pages/discover-more/related-projects/related-projects.md

```diff
@@ -55,6 +55,7 @@ This is a collection of third-party projects that extend Material-UI.
 - [redux-form-material-ui](https://github.com/erikras/redux-form-material-ui) Bindings for using Material-UI with [Redux Form](https://redux-form.com/).
 - [mui-rff](https://github.com/lookfirst/mui-rff) Bindings for using Material-UI with [React Final Form](https://final-form.org/react).
 - [data-driven-forms MUI mapper](https://github.com/data-driven-forms/react-forms/tree/master/packages/mui-component-mapper) Bindings for using Material-UI with [Data Driven Forms](https://data-driven-forms.org/).
+- [react-material-ui-stripe-payment-form](https://gitlab.com/angeloron/react-material-ui-stripe-payment-form) Bindings for using Material-UI with [Stripe payment](https://stripe.com)
```

Comment:
Do you think that you could host the project on GitHub? Also, I wonder, have you considered to expose low-level utils instead of high-level components? I have this problem in mind: #16037.

Reply:
Hosting on gitlab is fine. It shouldn't be important where this is hosted. There are political implications why you would or would not use github or gitlab.

Reply:
I added the project to github as well: https://github.com/angeloron/react-material-ui-stripe-payment-form.
As for low-level utils versus high-level components, I just wanted to provide a simple form and did not want to recreate libraries that already exist such as [react-stripe-elements](https://github.com/stripe/react-stripe-elements) or [react-stripe-js](https://stripe.com/docs/stripe-js/react). If you prefer I can suggest to have it in the "Showcase" instead of the "Related Projects" section.

Reply:
@eps1lon The incentive I see for using GitHub over GitLab is about leveraging past learning people have. The developers using Open Source are forced to learn and to get used to how GitHub works anyway. When going to GitLab, these developers might be lost. For instance, in my case, I don't understand how the UI works, I have never used it enough to learn it. These should give an idea of the scale of how much people know how GitHub works vs GitLab: https://www.similarweb.com/website/github.com?competitors=gitlab.com, about x20.

Hopefully, hosting the source on GitHub will translate into more feedback from users (issues, pull requests, stars) and overall higher quality software. At least, it's my argument.

---

@angeloron Thanks for the update. I don't think that we should link a high-level form, however, I think that if we could have a `material-ui-stripe` project, this would be really great. Something that people can then integrate with react-simple-form, formik, etc.

Reply:
@oliviertassinari It's a good idea of creating material-ui-stripe and I added it to one of my projects. The country I currently live in (Denmark) is coming out of confinement on Monday so I am going back to work and won't have that much time anymore.
Now should I anyway put https://github.com/angeloron/react-material-ui-stripe-payment-form as part of your "Showcase"?

Reply:
I don't think that a link in the related projects will bring in a lot of traffic. I don't think that a lot of developers are looking for this level of abstraction either. I would be in favor of waiting for the organic growth of your repository before linking it.

Reply:
I understand, makes sense. In the meantime I found [this](https://www.npmjs.com/package/material-ui-stripe-elements) which covers your idea of creating a "material-ui-stripe" project.
Then you did not answer on the "showcase section", do you think it would be worth to add it there.

Reply:
Thanks for looking.

For the showcase, we are looking for applications used in production. It wouldn't quality.